src/react-components/room/ChatSidebarContainer.js

```diff
@@ -267,6 +272,18 @@ export function ChatSidebarContainer({ scene, canSpawnMessages, presences, occup
         onChange={e => setMessage(e.target.value)}
         placeholder={placeholder}
         value={message}
+        isOverMaxLength={isOverMaxLength}
+        warning={
```

Comment:
I find this more readable than adding a `maxLength` prop instead since I only need the calculation in the component. However, I removed the `isOverMaxLength` prop in the `<ChatLengthWarning>` because it already has `messageLength` and `maxLength` everything it needs to calculate it. The extra prop was redundant.